apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V30__Create_preadjudicated_claims_tables.sql

```diff
@@ -0,0 +1,73 @@
+/*
+ * Creates the RDA FISS pre-adjudicated claims tables required by the application.
+ * https://confluenceent.cms.gov/pages/viewpage.action?spaceKey=MPSM&title=MVP+0.1+Data+Dictionary+for+Replicated+Data+Access+API
+ */
+
+/*
+ TODO - Determine whether we want to store tables in public schema or create a new one.
+ For now the table names are prefixed with PreAdj (pre-adjudicated).
+ */
+
+/*
+TODO - what tablespace should we use as a default?
+*/
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "PreAdjFissClaims"
+(
+    "dcn"               varchar(23) not null,
+    "hicNo"             varchar(12) not null,
+    "currStatus"        char(1)     not null,
+    "currLoc1"          char(1)     not null,
+    "currLoc2"          varchar(5)  not null,
+    "medaProvId"        varchar(13),
```

Comment:
"meda"? What's that short for?

Reply:
"Medicare", this is the Medicare Provider Number field

Reply:
Is that prefix coming from the upstream? I'm surprised by that versus something like `medi`

Reply:
Yeah its defined within FISS

Reply:
I'm glad I'm not the only one to have my silly typos preserved for perpetuity.